tools/cve/base-images.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Copyright 2022 Flant JSC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -Eeo pipefail
+shopt -s failglob
+
+if [[ "x$SEVERITY" == "x" ]]; then
+  SEVERITY="CRITICAL,HIGH"
+fi
```

Comment:
Why do you set SEVERITY inside if ? Script allows to run something like this 
```
SEVERITY=info base-images.sh
```
?
If yes, should we add some help info ?

Reply:
Ok, it makes sense. I will ad a comment.